pepper-apis/src/main/java/org/broadinstitute/ddp/housekeeping/schedule/CheckAgeUpJob.java

```diff
@@ -0,0 +1,106 @@
+import org.quartz.CronScheduleBuilder;
+import org.quartz.DisallowConcurrentExecution;
+import org.quartz.Job;
+import org.quartz.JobBuilder;
+import org.quartz.JobDetail;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.quartz.JobKey;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@DisallowConcurrentExecution
+public class CheckAgeUpJob implements Job {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CheckAgeUpJob.class);
+
+    public static JobKey getKey() {
+        return Keys.AgeUp.CheckJob;
+    }
+
+    public static void register(Scheduler scheduler, Config cfg) throws SchedulerException {
+        String schedule = ConfigUtil.getStrIfPresent(cfg, ConfigFile.CHECK_AGE_UP_SCHEDULE);
+        if (schedule == null || schedule.equalsIgnoreCase("off")) {
+            LOG.warn("Job '{}' is set to be turned off", getKey());
+            return;
+        }
+
+        JobDetail job = JobBuilder.newJob(CheckAgeUpJob.class)
+                .withIdentity(getKey())
+                .requestRecovery(false)
+                .storeDurably(true)
+                .build();
+        scheduler.addJob(job, true);
+        LOG.info("Added job '{}' to scheduler", getKey());
+
+        Trigger trigger = TriggerBuilder.newTrigger()
+                .withIdentity(Keys.AgeUp.CheckTrigger)
+                .forJob(getKey())
+                .withSchedule(CronScheduleBuilder
+                        .cronSchedule(schedule)
+                        .inTimeZone(TimeZone.getTimeZone(ZoneId.of("UTC")))
+                        .withMisfireHandlingInstructionFireAndProceed())
+                .startNow()
+                .build();
+        scheduler.scheduleJob(trigger);
+        LOG.info("Added trigger '{}' for job '{}' with schedule '{}'", trigger.getKey(), getKey(), schedule);
+    }
+
+    @Override
+    public void execute(JobExecutionContext context) throws JobExecutionException {
+        try {
+            LOG.info("Running job '{}'", getKey());
+            long start = Instant.now().toEpochMilli();
+            TransactionWrapper.useTxn(TransactionWrapper.DB.APIS, this::run);
+            long elapsed = Instant.now().toEpochMilli() - start;
+            LOG.info("Job '{}' completed in {} ms", getKey(), elapsed);
+        } catch (Exception e) {
+            LOG.error("Error while executing job '{}' ", getKey(), e);
+            throw new JobExecutionException(e, false);
+        }
+    }
+
+    private void run(Handle handle) {
+        AgeUpService service = new AgeUpService();
+        PexInterpreter interpreter = new TreeWalkInterpreter();
+        List<GovernancePolicy> policies = handle.attach(StudyGovernanceDao.class)
+                .findAllPolicies()
+                .collect(Collectors.toList());
+        for (var policy : policies) {
```

Comment:
Can you throw in this:
```
Collections.shuffle(policies);
```
The motivation here is that if there's a "bad apple" in the list, each run of the job will be in a different order, so it's less likely that we'll get "stuck" at one point in the list.  The try catch below is good too for tolerating errors and moving on to the next item.